docs/Collaboration guidelines.md

```diff
@@ -0,0 +1,51 @@
+Collaboration Guidelines
+===
+
+Be polite to each other
+---
+
+No one wants to wake up to a rude reply. Try to behave the best way you can, and show understanding when others are not. 
+
+When things escalate, try to use this document as a guideline on how to de-escalate them.
+
+Think of the project first
+---
+
+We're primarily here to make Reicast better.
+
+When discussing things in the tracker, forum or discord, try to think of the project first, and not of personal issues you might have with people.
+
+Good example: Constructive feedback is good because it makes contributors feel accepted, valued and safe. When people feel safe they are more comfortable explaining their opinion/viewpoint, and that helps make the project better
+
+Bad example: Emotionally abusing people is fun and makes my day better
+
+Avoid negativity, be realistic, give constructive feedback
+---
+
+Before replying, ask yourself the following questions.
+- How will the person I'm replying to feel after reading my comment?
+- Am I motivating them to get work done, or just spreading negativity?
+- Am I giving feedback that is valuable?
+
+Gracefully recover from missteps
```

Comment:
Another good point. You may also want to include "and don't be afraid to ask for help"